extensions/nsis-reference/src/index.tsx

```diff
@@ -1,6 +1,8 @@
-import { ActionPanel, Detail, Icon, List, PushAction } from "@raycast/api";
+import { ActionPanel, Detail, Icon, List, Action } from "@raycast/api";
+// @ts-expect-error Cannot find module '@nsis/docs' or its corresponding type declarations.ts (2307)
```

Comment:
I'd like to address this in `@nsis/docs`, so hopefully we can get rid off this comment